component/profile/reader/file/creator.go

```diff
@@ -22,12 +22,18 @@ import (
 )
 
 // nolint: gochecknoglobals
-var signatureKeyTypeMap = map[vcsverifiable.SignatureType]string{
+var signatureTypeToDidVerificationMethod = map[vcsverifiable.SignatureType]string{
 	vcsverifiable.Ed25519Signature2020:        crypto.Ed25519VerificationKey2020,
 	vcsverifiable.Ed25519Signature2018:        crypto.Ed25519VerificationKey2018,
 	vcsverifiable.JSONWebSignature2020:        crypto.JSONWebKey2020,
 	vcsverifiable.EcdsaSecp256k1Signature2019: crypto.EcdsaSecp256k1VerificationKey2019,
 	vcsverifiable.BbsBlsSignature2020:         crypto.Bls12381G1Key2020,
+	// JWT
+	vcsverifiable.EdDSA:  crypto.JSONWebKey2020,
+	vcsverifiable.ES256K: crypto.JSONWebKey2020,
+	vcsverifiable.ES256:  crypto.JSONWebKey2020,
+	vcsverifiable.ES384:  crypto.JSONWebKey2020,
+	vcsverifiable.PS256:  crypto.JSONWebKey2020,
```

Comment:
Please pay extra attention on this part of code - it's mapping between signature type and did verification method type. It's used [here](https://github.com/trustbloc/vcs/pull/827/files#diff-6e7fd18916da9184e146724584d2202fbe95e42bf84f7900252d82dcd94131d8L208). 
I'm not sure whether I'm right.